docs/designers-developers/developers/tutorials/create-block/submitting-to-block-directory.md

```diff
@@ -0,0 +1,102 @@
+- Clearly describe your block
+- Add Keywords for all contexts
+- Choose the right category
+
+### Name your block based on what it does
+
+Users typically search the block directory within the Block Editor and do so in the context of a task. For example, when building their post, a user may search the Block Directory for an “image gallery”. Naming your block accordingly will help the Block Directory surface it when it's needed.
+
+**Not So Good**: WebTeam5 Image Works
+**Good**: Responsive Image Slider by WebTeam5
+
+**Question: What happens when there are multiple blocks with similar names?**
+Try your best to make your block's name functional and unique to make it stand out. Look for applicable synonyms or include a prefix if necessary.
+
+### Clearly describe your block
+
+The description really helps to communicate what your block does.The quicker a user understands how your block will help them, the more likely it is a user will use your block. Users will be reading your block's description within the Block Editor where space can be limited. Try to keep it short and concise.
+
+**Not So Good**: The best way to show images on your website using jQuery and CSS.
+**Good**: An easy to use, responsive, Image gallery block. 
+
+**Tip**: It’s not about marketing your block, in fact we want to avoid marketing in blocks. You can read more about it in the [plugin guidelines]. Stick to being as clear as you can. The block directory will provide metrics to let users know how awesome your block is!
+
+### Add Keywords for broader context
+
+Keywords add extra context to your block and make it more likely to be found in the inserter. 
+
+Examples for an Image Slider block:
+- slider
+- carousel
+- gallery
+
+[Read more about keywords.](https://github.com/WordPress/gutenberg/blob/master/docs/rfc/block-registration.md#keywords)
+
+### Choose the right category
+
+The Block Editor allows you to indicate the category your block belongs in, making it easier for users to locate your block in the menu.
+
+**Possible Values**:
+- text
+- media
+- design
+- widgets
+- embed
+
+[Read more about categories.](https://github.com/WordPress/gutenberg/blob/master/docs/rfc/block-registration.md#category)
+
+Wondering where to input all this information? Read the next section :)
+
+## Step 2: Analyze your plugin
+Each block in your plugin should have a corresponding `block.json` file. This file provides the Block Directory important information about your block. Along with being the place to store contextual information about your block like the: `name`, `description`, `keywords` and `category`, the `block.json` file stores the location of your block’s files.
+
+Block plugins submitted to the Block Directory can contain mutliple blocks only if they are children of a single parent/ancestor. There should only be one main block. For example, a list block can contain list-item blocks. Children blocks must set the `parent` property in their `block.json` file.
+
```

Comment:
Block plugins submitted to the Block Directory can contain **multiple** (typo) blocks only if they are children of a single parent/ancestor. 

Rephrasing.
Block plugins submitted to the Block Directory should have one main block, but it can contain multiple child blocks. 
For example, a list block can contain list-item child blocks. Child blocks must set the `parent` property in their `block.json` file.